arangod/RestHandler/RestDocumentHandler.cpp

```diff
@@ -277,9 +277,10 @@ futures::Future<futures::Unit> RestDocumentHandler::insertDocument() {
   // track request only on leader
   if (opOptions.isSynchronousReplicationFrom.empty() &&
       ServerState::instance()->isDBServer()) {
-    _activeTrx->state()->trackRequest(*_activeTrx->resolver(), _vocbase.name(),
-                                      cname, _request->value("user"),
-                                      AccessMode::Type::WRITE, "insert");
+    _activeTrx->state()->trackShardRequest(
+        *_activeTrx->resolver(), _vocbase.name(), cname,
+        _request->value(StaticStrings::UserString), AccessMode::Type::WRITE,
+        "insert");
```

Comment:
Minor: We could think about introducing enums or StaticStrings here (Must not happen here, could be a follow up pr)

Reply:
If this comment refers to the last parameter (`"insert"`), this is just an informational parameter that I used for debugging. It is not used for storing anything inside the metrics.